impress-org/givewp
# Feature: add FormMeta and PdfSettings steps to the form migration

## Description



This PR adds a new step in the form migration to catch all form meta keys from the old form into the new one.

This PR also adds a new step for the PDF settings so we can make sure that when the user edits a migrated form using the Form Builder, everything works as in the v2 forms.

~~It also adds a new `givewp_form_builder_pdf_settings_default_values` filter, allowing us to set default values when these settings are empty. More details here: https://github.com/impress-org/give-pdf-receipts/pull/253~~

## Affects



Form meta keys for v3 forms and pdf settings in v3 forms.

## Testing Instructions



1. Create a v2 form and change the settings for the PDF receipt in this form
2. Migrate the v2 form from the previous step to a v3 form
3. Check the PDF setting of the v3 form, it should reflect the exact same settings from the v2 form
4. To check if other meta keys (not related to the pdf settings) also were migrated, check it directly on the database on the `wp_give_formmeta` table

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205615499105028
  - https://app.asana.com/0/0/1205573027035196
  - https://app.asana.com/0/0/1205604772387034
